support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -259,6 +259,35 @@ function getAmountsTestVariant(
 		}
 	};
 
+	// Is the country in the list for contributions only checkout?
+	// This relies on the existence of an amounts test with a specific name
+	// It could be done using the countriesAffectedByVATStatus list
+	// but this would need the list and the amounts test to be in sync
+	// and this way simplifies testing as it is all set up in the RRCP
+	const contribOnlyTestName = 'VAT_COMPLIANCE';
```

Comment:
just wondering if we should line up the name and the const, unless there's a reason for the difference?

Reply:
That's a good question. The difference is due to how we speak about this project. The contributions only checkout is part of the VAT compliance project.
This amounts test is used to show the contributions only checkout, which is why I named the const this way as it seemed more specific. The amounts test itself was created back in November 2023, and predates the use of the term "contributions only" in this project.

Happy to rename the const if you think it's confusing.

Reply:
that's fine, I was mainly wanted to check it was considered (and not a copy and paste error!)